sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 1.0.0-preview.4 (Unreleased)
 ### Breaking changes
 - Replaced `TextAnalyticsApiKeyCredential` with `AzureKeyCredential`.
-- Rename all input parameters `inputText` to `document`, and `inputs` to `documents`.
+- Renamed all input parameters `inputText` to `document`, and `inputs` to `documents`.
```

Comment:
Shouldn't we reflect the change in the type hierarchy here?

Reply:
Good point. I was thinking about that and wasn't sure.
So should I include it even if it is not a breaking change?

Reply:
Yes, we aren't limited only to breaking changes.

Reply:
Thank you! I will update